packages/components/flag/src/components/osds-flag/core/controller.ts

```diff
@@ -31,7 +27,7 @@ class OdsFlagController {
    * @param isVisible - is the element to load is visible or not
    * @param isBrowser - is the context is browser one or not
    */
-  load(isVisible: boolean, isBrowser = true): Promise<string | void> {
+  async load(isVisible: boolean, isBrowser = true): Promise<string | void> {
```

Comment:
No await used

Reply:
It's just a best practice to show when a function returns a Promise